app/boards/shields/splitkb_aurora_corne/splitkb_aurora_corne.dtsi

```diff
@@ -20,6 +20,7 @@
 
     chosen {
         zephyr,display = &oled;
+        zmk,physical-layout = &foostan_corne_6col_layout;
```

Comment:
I'm fine with this now that we've got warning + fallback for folks with a chosen transform set in existing config repos.